### ISSUE TITLE
Error executing vim.schedule lua callback: ...vim/0.10.0/share/nvim/runtime/lua/vim/lsp/diagnostic.lua:237: attempt  to get length of local 'diagnostics' (a nil value)

### ISSUE THREAD

**Issue by dcy** [lasorda/protobuf-language-server#14]
When i open xxx.proto, or in insert mode. I will get this error:
```shell
Error executing vim.schedule lua callback: ...vim/0.10.0/share/nvim/runtime/lua/vim/lsp/diagnostic.lua:237: attempt
 to get length of local 'diagnostics' (a nil value)
stack traceback:
        ...vim/0.10.0/share/nvim/runtime/lua/vim/lsp/diagnostic.lua:237: in function 'handle_diagnostics'
        ...vim/0.10.0/share/nvim/runtime/lua/vim/lsp/diagnostic.lua:293: in function 'on_publish_diagnostics'
        ...aiyun/.local/share/nvim/lazy/go.nvim/lua/go/lsp_diag.lua:38: in function 'handler'
        .../neovim/0.10.0/share/nvim/runtime/lua/vim/lsp/client.lua:1005: in function ''
        vim/_editor.lua: in function <vim/_editor.lua:0>
```

**Comment by BrotifyPacha** [comment]
This should fix it #15

**Comment by BrotifyPacha** [comment]
This should fix it #15
